man/pam_systemd.xml

```diff
@@ -314,7 +314,7 @@ auth      required   pam_deny.so
 
 account   required   pam_nologin.so
 -account  sufficient pam_systemd_home.so
-account   sufficient pam_unix.so
+account   sufficient pam_unix.so no_pass_expiry
```

Comment:
hmm, but the "sufficient" in the pam_systemd_home line means that this line isn't even tested anymore, hence this should not be needed